frontend/src/components/checkbox/checkbox-styles.ts

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+export const CheckboxWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
```

Comment:
flex-direction is row by default so this can be omitted.

Reply:
Yes yes, I just left it in case I needed to switch to column. But in the end I deleted it as you said